server/boot/customRoutes.js

```diff
@@ -4,75 +4,135 @@ module.exports = function (app) {
-    // const tomorrow = moment().utc().add(1, 'days').startOf('day').toISOString(); 
-    // const yesterday = moment().utc().subtract(1, 'days').endOf('day').toISOString();
     const dayStart = moment().utc().startOf('day').toISOString();
     const dayEnd = moment().utc().endOf('day').toISOString();
-
+    
     Slot.find({
       "where": { 
         date: 
           { between: [dayStart, dayEnd] }, 
           storeId: storeId
       }, include: 'bookings'})
       .then(slots => {
-        if (!slots) {
-          res.json({ "error": "No slots created for this date and storeId yet."})
+        if (slots.length < 1) {
+          return res.json({ "error": "No slots created for this date and storeId yet."})
         } else { 
         const avail = slots.filter((booking) => {
             let numBookings = booking.bookings().length; 
             let maxPeoplePerSlot = booking.maxPeoplePerSlot;
             return numBookings < maxPeoplePerSlot;
         });
-        res.json(avail);
+        return res.json(avail);
       }})
       .catch(err => console.log(err));
   })
 
-  router.get("/api/generateSlots", (req, res) => {
-    //Generate new slots for today.
-    const { storeId } = req.body; 
-    Store.findById(storeId)
-      .then(store => { 
-        const { id, openingHour, closingHour, slotDuration, maxPeoplePerSlot } = store; 
-        const newSlots = generateTimeslots(slotDuration, openingHour, closingHour, id, maxPeoplePerSlot);
-        Slot.create(newSlots)
-          .then(newSlots => { 
-            res.json(newSlots)
-          })
-          .catch(err => console.log(err))
-      })
-      .catch(err => console.log(err));
+  // Auth Routes  
+  router.get("/auth/google/", passport.authenticate("google", {
+    scope: ['profile', 'email']
+ }))
+
+  router.get("/auth/google/redirect", passport.authenticate('google'), (req, res) => {
+    //User is logged in at this point. Should redirect user to their home page.
+    res.redirect("/")
+  })
+
+  router.get("/auth/getUser", (req, res) => {
+    res.json(req.user);
+  })
+
+  router.post("/auth/logout", (req, res) => {
+      req.logout();
+      res.redirect("/")
+  })
+
+  //Set up Nodemailer 
+  const transporter = nodemailer.createTransport({
+    service: 'gmail', 
+    auth: { 
+      user: 'teamsafeslot@gmail.com', 
+      pass: keys.nodeMailer.key
+    }
+  })
+
+  //Mail route 
+  router.post('/api/mail', (req, res) => {
+    //Need to add QR code to email 
+    const { toAddress, slotTime, slotDate, slotId, storeName } = req.body; 
+    const mailOptions = {
+      from: 'teamsafeslot@gmail.com', 
+      to: toAddress, 
+      subject: 'Your Safeslot Reservation', 
+      html: `Your reservation at ${storeName} is for ${slotDate} at ${slotTime}. Please show the store your QR code at the door.`
+    }
+  
+    transporter.sendMail(mailOptions, (err, info) => {
+     if (err) {
+       console.log(err)
+     } else {
+       console.log(info)
```

Comment:
since you're logging both the error and info, you can just do `console.log(err, info)`